releasenotes/index.shtml

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+ <title>
+ JMRI: History - differences between the various applications
```

Comment:
Is this the right title for this page?

Reply:
Thanks. That gave away its origin a bit too obviously